hw3/01-characters.js

```diff
@@ -1,4 +1,24 @@
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
+const section = document.querySelector('section');
 
-fetch(url);
+fetch(url)
+  .then((response) => response.json())
+  .then((data) => {
+    for (const character of data) {
+      const card = document.createElement('card');
+      const image = document.createElement('img');
+      image.src = character.imageUrl;
+      image.height = '200';
```

Comment:
Worked as well on mobile as it did on web. Might consider margin for the images.